skins/laika/src/store/track_form_errors_plugin.ts

```diff
@@ -0,0 +1,40 @@
+import { MutationPayload, Store } from 'vuex';
+import { mutation as mutationName } from '@/store/util';
+import { NS_ADDRESS, NS_MEMBERSHIP_ADDRESS } from '@/store/namespaces';
+import { trackFormValidationErrors } from '@/tracking';
+import { VALIDATE_INPUT } from '@/store/address/mutationTypes';
+import { Validity } from '@/view_models/Validity';
+
+export const trackFormErrorPlugin = ( store: Store<any>, formName: string ) => {
+
+	let formNameToStoreNameLookUp: { [key: string]: string} = {
+		'donation_form': 'address',
```

Comment:
These strings now accidentally correspond with the from name constants you provided in the entry points. To kake the meaning/semantics more clear, I think you should move those form names as consteants here, import them in the entry points and use them in the lookup table like this:

```
	const formNameToStoreNameLookUp: { [key: string]: string} = {
		[FORM_DONATION]: 'address',
                [FORM_ADDRESS_UPDATE]: 'address',
                [FORM_MEMBERSHIP]: 'membership_address'
};
```

Reply:
in his PR Abban is using those form names in the entry points as well, so I shouldn't really move them..?
https://github.com/wmde/FundraisingFrontend/pull/1741/files